docs/index.md

```diff
@@ -33,22 +33,26 @@ about_list:
 # Resource links
 about_resources_title: Resources
 
+about_system_diagram: assets/img/system-diagrams/system-diagram.svg
+about_system_diagram_caption: System Overview
+about_system_link: system-overview
+
 about_resources:
   - name: Gitter
     link: https://gitter.im/DigitalSlideArchive/HistomicsTK
-    icon: fab fa-gitter #assets/img/resource-icons/icon_gitter.png
+    icon: fab fa-gitter
 
   - name: Discourse
     link: https://discourse.girder.org/c/histomicstk
-    icon: fab fa-discourse #assets/img/resource-icons/icon_discourse.png
+    icon: fab fa-discourse
 
   - name: Jupyter
     link: https://digitalslidearchive.github.io/HistomicsTK/examples.html
-    icon: fas fa-book #assets/img/resource-icons/icon_jupyternb.png
+    icon: fas fa-book
 
   - name: Youtube
-    link: https://www.youtube.com/channel/UCe9RJmSdEJLTWkRhSOIVAlg
-    icon: fab fa-youtube #assets/img/resource-icons/icon_youtube.png
+    link: https://www.youtube.com/channel/UCJVNVQXivYPDb4goNU4tK3A
```

Comment:
We had content in youtube on two channels; `https://www.youtube.com/channel/UCe9RJmSdEJLTWkRhSOIVAlg` is the preferred link.

Reply:
thanks for letting me know, fixed it back. 
ded94ff